lego/apps/surveys/permissions.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import permissions
+
+from lego.apps.events import constants
+from lego.apps.events.models import Registration
+from lego.apps.permissions.constants import EDIT
+from lego.apps.permissions.permissions import PermissionHandler
+
+
+class SurveyPermissionHandler(PermissionHandler):
+    pass
+
+
+class SubmissionPermissions(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        from lego.apps.surveys.models import Survey
+        survey = Survey.objects.get(id=view.kwargs['survey_pk'])
+        event = getattr(survey, 'event')
+        user = request.user
+        user_attended_event = Registration.objects.filter(
+            event=event.id, user=user.id, presence=constants.PRESENT).exists()
+
+        if user.has_perm(EDIT, obj=Survey):
+            return True
+        if view.action in ['create']:
+            return user_attended_event
```

Comment:
Do we want to add a check here to make sure each user can only answer once per survey?

Reply:
Submission already has `unique_together = (survey, user)`, but perhaps checking for it here would provide a better error message?